src/grt/grt-avhpi.adb

```diff
@@ -398,6 +526,7 @@ package body Grt.Avhpi is
                Bt : constant Ghdl_Rtin_Type_Array_Acc :=
                  To_Ghdl_Rtin_Type_Array_Acc (Atype.Basetype);
             begin
+               -- I don't understand what's happening here.
```

Comment:
Ah.  An array type declaration like `type my_vec is array (0 to 7) of bit` is represented as an anonymous unconstrained array type declaration `type anon_my_vec is array (integer range <>) of bit` followed by a subtype declaration `subtype my_vec is anon_my_vec(0 to 7)`.  So we need to ignore the anonymous array type declaration and pretend that the subtype declaration is a type declaration.